renderdoc/driver/gl/gl_pixelhistory.cpp

```diff
@@ -75,6 +160,24 @@ bool PixelHistorySetupResources(WrappedOpenGL *driver, GLPixelHistoryResources &
                              desc.msSamp, 1);
   driver->glFramebufferTexture(eGL_FRAMEBUFFER, eGL_DEPTH_STENCIL_ATTACHMENT, resources.dsImage, 0);
 
+  // Allocate stencilFragmentCountFrameBuffer
+  driver->glGenFramebuffers(1, &resources.stencilFragmentCountFrameBuffer);
+  driver->glBindFramebuffer(eGL_FRAMEBUFFER, resources.stencilFragmentCountFrameBuffer);
+
+  // Allocate stencilFragmentCountImage
+  driver->glGenTextures(1, &resources.stencilFragmentCountImage);
+  driver->glBindTexture(eGL_TEXTURE_2D, resources.stencilFragmentCountImage);
+  driver->CreateTextureImage(resources.stencilFragmentCountImage, eGL_DEPTH32F_STENCIL8, eGL_NONE,
+                             eGL_NONE, eGL_TEXTURE_2D, 2, desc.width >> sub.mip,
+                             desc.height >> sub.mip, 1, desc.msSamp, 1);
+
+  driver->glReadBuffer(eGL_NONE);
+  driver->glDrawBuffer(eGL_NONE);
+
+  rdcstr fs =
+      GenerateGLSLShader(GetEmbeddedResource(glsl_pixelhistory_primid_frag), ShaderType::GLSL, 460);
```

Comment:
460 seems like a magic number here, maybe we want to put it in a variable that explains its meaning?